dvc/fs/dvc.py

```diff
@@ -571,7 +571,7 @@ def _get(  # noqa: C901, PLR0912
 
         os.makedirs(lpath, exist_ok=True)
         for d in _dirs:
-            os.mkdir(d)
+            os.mkdir(d, exist_ok=True)
```

Comment:
There is no `exist_ok=True` in `os.mkdir`. That is in `os.makedirs`.


This should either be:

```suggestion
            try:
                os.mkdir(d)
            except OSError:
                if not os.path.isdir(d):
                    raise
```

or,

```suggestion
            os.makedirs(d, exist_ok=True)
```


I'd prefer the first one as `makedirs` calls `os.path.exists()` on all the parents of the path.

Reply:
Any reason why you did not go with the first option?

Reply:
Less verbose, atomicity and clarity.